### ISSUE TITLE
Ability to specify the order of any pack no matter who sends them

### ISSUE THREAD

**Issue by Phoenix616** [Phoenix616/ResourcepacksPlugins#83]
#### Is your feature request related to a problem? Please describe.

When a server sends a pack which should be between two that the user already had applied before (e.g. from a proxy) then this isn't really supported right now.

#### Describe the solution you'd like

A way to configure it, e.g. with a priorities list.

#### Describe alternatives you've considered

Maybe merging or trying to define a list per-server/world of which order all packs should have.

#### Additional context

Only applies to 1.20.3+

**Comment by SinonCute** [comment]
I have a similar problem with ItemsAdder. I have three servers using the same IA, and I disabled self-hosting for two of them, leaving only one to host the pack. The plugin works great when I join the two servers with self-hosting disabled, but when I join the main server that hosts the pack, it duplicates the pack from Velocity and one from the server. I believe this issue can be solved by ignoring the server or setting a priority between them.

**Comment by Phoenix616** [comment]
> I have a similar problem with ItemsAdder. I have three servers using the same IA, and I disabled self-hosting for two of them, leaving only one to host the pack. The plugin works great when I join the two servers with self-hosting disabled, but when I join the main server that hosts the pack, it duplicates the pack from Velocity and one from the server. I believe this issue can be solved by ignoring the server or setting a priority between them.

This issue is about this plugin sending packs, not other plugins (catching those isn't really supported). You should disable sending the pack in ItamsAdder and only use my plugin if you want it to be global.

**Comment by SinonCute** [comment]
When I generate a new file using /iazip, I use the endpoint host file of ItemsAdder to link it inside the plugin. Is there a way to ignore the send event from the backend to the player?

**Comment by Phoenix616** [comment]
> When I generate a new file using /iazip, I use the endpoint host file of ItemsAdder to link it inside the plugin. Is there a way to ignore the send event from the backend to the player?

No. Just disable the sending in ItemsAdder. I'm sure there is a way, lots of people use it with my plugins. If you can't figure out how then I suggest contacting their support, they will know best how it works.

**Comment by SinonCute** [comment]
Thank you, I figured out.
